webapp/src/index.tsx

```diff
@@ -545,7 +575,9 @@ export default class Plugin {
             const currentChannelId = getCurrentChannelId(store.getState());
             if (currChannelId !== currentChannelId) {
                 currChannelId = currentChannelId;
-                fetchChannelData(currChannelId);
+                fetchChannelData(currChannelId).then(() => {
+                    followThread(currChannelId, getCurrentTeamId(store.getState()));
```

Comment:
What's the reason for following a thread on channel switch?

Reply:
that is a missunderdstanding on that part of the code, I thought it was switching from one call to other, not channels. I'll remove this.